packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -149,8 +153,23 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
     if (this.props.parentRef) {
       this._fitParentContainer();
     }
+
+    const margin = { ...this.margins };
+    if (this.props.showYAxisLables) {
+      const maxYAxisLabelLength = calculateLongestYAxisLabel(points, this.props.theme!);
+      if (this.state.startFromX !== maxYAxisLabelLength) {
+        this.setState({
+          startFromX: maxYAxisLabelLength,
+        });
+      }
+      margin.left! += this.state.startFromX;
```

Comment:
Have you tested the new chart for RTL?

Reply:
Now changes are done for RTL as well now, it is working properly.